src/extension/iSamples_results.js

```diff
@@ -40,7 +38,7 @@ class iSamples_Result extends React.Component {
     return (
       <li className={cx({ "list-group-item": bootstrapCss })} onClick={() => this.props.onSelect(doc)}>
         <ul>
-          {fields.filter((field) => field.field !== "*").map((field, i) =>
+          {fields.filter((field) => field.field !== "*" && field.collapse !== true).map((field, i) =>
```

Comment:
So this is the piece that prevents the collapsed fields from showing in the results?